bindings/java/java/src/main/java/org/ethereum/evmc/EvmcVm.java

```diff
@@ -92,14 +94,14 @@ public String version() {
    *
    * <p>This is a mandatory method and MUST NOT be set to NULL.
    */
-  native void destroy(ByteBuffer nativeVm);
+  private static native void destroy(ByteBuffer nativeVm);
 
   /**
    * Function to execute a code by the VM instance.
    *
    * <p>This is a mandatory method and MUST NOT be set to NULL.
    */
-  native void execute(
+  private static native void execute(
```

Comment:
Why is this static?

Reply:
Because it only exists once, it is the native library which can only be loaded once.

Reply:
OK, if it's purely functional, that may work.